File: problemas/iniciante/2543/2543.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.trim().split('\n');

let p = 0;
while (p < lines.length) {
    let [N, I] = lines[p++].trim().split(' ').map(x => parseInt(x));

    let gameplays = 0;
    for (let i = 0; i < N; ++i) {
        let [identificador, j] = lines[p++].trim().split(' ').map(x => parseInt(x));

        if (identificador === I && j === 0) {
            ++gameplays;
        }
    }

    console.log(gameplays);
}

File: problemas/iniciante/2543/2543.py
while True:
    try:
        N, I = [int(x) for x in input().strip().split(' ')]

        gameplays = 0
        for _ in range(N):
            identificador, j = [int(x) for x in input().strip().split(' ')]

            if(identificador == I and j == 0):
                gameplays += 1

        print(gameplays)
    except EOFError:
        break
